### PR TITLE
Citing GA4GH Documentation - PR

### PR DESCRIPTION
**Pull request by andy-yates-hdr**
PR merge to bring in Citing GA4GH Documentation proposal into the TASC repository with steps towards resolving #39

### CODE REVIEW

File: drafts/Citing GA4GH Documentation.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+
+
+### DOI Request Process
+
+Individuals responsible for a documentary output requiring a DOI (output leads) MUST submit a DOI request through their originating group co-leads (e.g. Work Stream, NIF, GIF, or TASC leads). These co-leads MUST define the metadata associated with the DOI request, including the `<std_designator>`, in collaboration with the output leads. The completed request MUST be submitted to the GA4GH Technical Team together with all required metadata (see *Metadata Management* for the complete list). The Technical Team MUST verify that the request is legitimate and appropriate prior to DOI creation. Where uncertainty exists regarding eligibility or governance, the Technical Team MAY escalate the request to the Chief Product Officer. Following DOI registration, the Technical Team MUST provide the DOI to the GA4GH Communications Team for publication through appropriate GA4GH channels. Automated versions of this process are permitted so long as the ethos of this flow is retained.
```
Comment (rrfreimuth) [inline drafts/Citing GA4GH Documentation.md]: This paragraph uses "originating group" with examples of WS, NIF, GIF, or TASC. Most other instances refer only to "work stream". I prefer the generalized term as it is more inclusive and likely to age more elegantly. I suggest using "originating group" throughout.

File: drafts/Citing GA4GH Documentation.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+- [Considerations](#considerations)
+- [References](#references)
+- [Contributors](#contributors)
+- [Notes](#notes)
+
+## Recommendation
+
+For significant GA4GH documentary outputs (e.g. products, standards, protocols, technical specifications, white papers, and selected blog posts) requiring a persistent identifier, Digital object identifiers (DOI) MUST use the GA4GH prefix `10.59756` resulting in DOIs with the following format `https://doi.org/10.59756/SUFFIX`. Registration requests MUST be submitted to the originating group's co-leads (e.g. work stream, NIF, GIF, TASC) and then to the Technical Team unless another automated minting method has been created. DOIs MUST be registered using the GA4GH approved Registration Agency, currently Crossref. DOI suffixes MUST be an obfuscated random string with no semantic meaning. The *Detailed Guidance* lists the minimal metadata that MUST be provided, which include the `<std_designator>` metadata field, pointing to the reference Work Stream.
+
+The `<std_designator>` metadata field SHOULD follow the format `<WORK_STREAM_ACRONYM>_<DOCUMENT_NAME>`.
```
Comment (rrfreimuth) [inline drafts/Citing GA4GH Documentation.md]: Suggest defining `<std_designator>` as `<ORIGINATING_GROUP_ACRONYM>_<DOCUMENT_NAME>`

File: drafts/Citing GA4GH Documentation.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+The Technical Team maintains a register of assigned suffixes to avoid clashing DOIs. DOIs MAY be created for any GA4GH digital output and for any version of a digital output. Newly approved products SHOULD request and assign a DOI to selected digital materials, including any specific versions. DOIs MUST NOT be minted for transitional or ephemeral materials e.g. a draft specification.
```
Comment (rrfreimuth) [inline drafts/Citing GA4GH Documentation.md]: Is there utility in expanding the register to include `WORK_STREAM_ACRONYM`/`ORIGINATING_GROUP_ACRONYM` and `DOCUMENT_NAME` to prevent clashes, maintain consistency, and enable reverse lookup?

File: drafts/Citing GA4GH Documentation.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+
+### Metadata Management
+
+Metadata associated with a DOI MUST accurately describe the referenced document at the time of registration. Metadata SHOULD be updated when document locations or other deposited metadata elements change. Where a document is relocated, the DOI MUST continue to resolve to the authoritative version of the document. Changes in document location MUST NOT require minting a new DOI unless a new citable object is being created.
+
+The complete list of required metadata follows. Fields marked *"to be specified"* MUST be supplied by the requester:
+
+- Contributor → Organization: to be specified (usually a GA4GH Work Stream)
+- Title: to be specified
+- Approval Date (month, day, year): to be specified
```
Comment (rrfreimuth) [inline drafts/Citing GA4GH Documentation.md]: Which approval is this? (e.g., of the product or of the DOI mint request)

File: drafts/Citing GA4GH Documentation.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+- Publisher Name: Global Alliance for Genomics and Health
+- Publisher Place: to be specified
```
Comment (rrfreimuth) [inline drafts/Citing GA4GH Documentation.md]: Is this a geographic location or a URL?

File: drafts/Citing GA4GH Documentation.md [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+- \[CROSSREF_RECORDS\] \- [Crossref Supported Record Types](https://www.crossref.org/documentation/schema-library/markup-guide-record-types/)
+- \[TASC-ADR-001\] - [TASC-ADR-001: Use Crossref as the DOI Registration Authority for GA4GH](../adr/TASC-ADR-001.md)
+- \[TASC-ADR-002\] - [TASC-ADR-002: Separate Citation Identifiers from Machine-Readable Standard Identifiers](../adr/TASC-ADR-002.md)
+- \[TASC-ADR-003\] - [TASC-ADR-003: Use Opaque DOI Suffixes](../adr/TASC-ADR-003.md)
+- \[TASC-ADR-004\] - [TASC-ADR-004: Use Standard Designators for DOI Metadata Classification](../adr/TASC-ADR-004.md)
+- \[TASC-ADR-005\] - [TASC-ADR-005: Do Not Extend GA4GH DOIs to Datasets or Other Research Resources](../adr/TASC-ADR-005.md)
+
+## Contributors
+
+[People who contributed to the development of this recommendation, including reviewers and discussion participants.]
```
Comment (rrfreimuth) [inline drafts/Citing GA4GH Documentation.md]: Are TASC members considered reviewers or discussion participants?

File: adr/TASC-ADR-002.md [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+
+### Negative
+
+- ❌ Multiple identifier systems may coexist
+- ❌ Additional governance may be required
+
+### Risks & Mitigations
+
+**Risk:** Users become confused by multiple identifiers
+- **Mitigation:** Clearly document the purpose and scope of each identifier type
```
Comment (rrfreimuth) [inline adr/TASC-ADR-002.md:67]: Where should this documentation exist? (e.g., product level, TASC, or tech team?)

File: adr/TASC-ADR-003.md [hunk 8]
```diff
@@ -0,0 +1,96 @@
...
+
+### Negative
+
+- ❌ DOI identifiers are less intuitive for human readers
+- ❌ Administrative systems must rely on metadata rather than identifier inspection
+
+### Risks & Mitigations
+
+**Risk:** Users attempt to infer meaning from DOI values
+- **Mitigation:** Publish guidance stating that meaning is carried through metadata and standard designators rather than the DOI suffix
```
Comment (rrfreimuth) [inline adr/TASC-ADR-003.md:82]: The registry should be listed as a mitigation strategy, if it is indexing the suffixes

File: drafts/TASC-ADR-004.md [hunk 9]
```diff
@@ -0,0 +1,95 @@
...
+
+GA4GH will require all DOI registrations to include a standard designator (`std_designator`) that identifies the originating Work Stream and the associated documentary output. The standard designator will be used as a metadata classification mechanism and will not form part of the DOI itself.
+
+**Key Points:**
+
+- Every DOI registration MUST include a `std_designator`
+- Standard designators MUST follow the format:
+
+  ```text
+  <WORK_STREAM_ACRONYM>_<DOCUMENT_NAME>
```
Comment (rrfreimuth) [inline drafts/TASC-ADR-004.md]: See comment re: `ORIGINATING_GROUP_ACRONYM`

File: adr/TASC-ADR-004.md [hunk 10]
```diff
@@ -0,0 +1,95 @@
...
+- ✅ Enables future automation and metadata analysis
+- ✅ Provides human-readable context independent of DOI identifiers
+
+### Negative
+
+- ❌ Requires governance of naming conventions
+- ❌ Requires coordination across Work Streams
+- ❌ Existing registrations may require metadata updates if conventions evolve
+
+### Risks & Mitigations
```
Comment (rrfreimuth) [inline adr/TASC-ADR-004.md:88]: Documenting naming conventions and indexing the subcomponents used would help mitigate governance and coordination challenges. Also, since all requests come through the tech team, they could be responsible for checking proposed designators.

File: adr/TASC-ADR-005.md [hunk 11]
```diff
@@ -0,0 +1,74 @@
...
+2. **Do not extend the GA4GH DOI mechanism to datasets; leave dataset identification out of scope**
+   - ✅ Keeps the GA4GH DOI program focused and operationally manageable
+   - ✅ Avoids duplicating existing dataset persistent-identifier infrastructure
+   - ❌ GA4GH datasets remain reliant on whatever identifier scheme their hosting repository provides
+
+---
+
+## Decision
+
+GA4GH will not use, or extend, its DOI mechanism to serve as the persistent identifier scheme for datasets, tools, or other research resources. The scope of GA4GH-minted DOIs is restricted to documentary outputs.
```
Comment (rrfreimuth) [inline adr/TASC-ADR-005.md:41]: We may want to clarify the use of "datasets" here. Products (code) are in scope, and mature releases may contain a test suite that may include a test data set. It would be logical to assume the test suite (including supporting data) is included in the overall "product" and therefore in scope for the DOI, but there may be some edge cases where the distinction is less clear.

**Review by rrfreimuth — COMMENTED** [review]
Great job! I found a few areas where a minor modification could add clarity.

**Review by melissacline — APPROVED** [review]
In Citing GA4GH Documentation, in the opening paragraph, the comments about creating DOIs whose suffixes are obfuscated is confusing in context, since it's not apparent at that point _why_ that's desirable.  My suggestion is to replace it with something that's positive and concrete, such as "encodes no fragile metadata".

One more small point is that the links at the bottom of Citing GA4GH Documentation are currently broken.

File: drafts/Citing GA4GH Documentation.md [hunk 12]
```diff
@@ -0,0 +1,159 @@
...
+
+DOI suffixes MUST be generated as opaque identifiers and MUST NOT encode semantic or document-specific meaning. Human-readable information SHOULD be represented through deposited metadata rather than through the DOI structure itself.
+
+### Standard Designator Assignment
+
+Each DOI registration MUST include a `<std_designator>` metadata field identifying the originating Work Stream and document context. The `<std_designator>` value MUST follow the format:
+
+
+```text
+<WORK_STREAM_ACRONYM>_<DOCUMENT_NAME>
```
Comment (monicacecilia) [inline drafts/Citing GA4GH Documentation.md]: Either here or above in line 34, include a link to the latest list of WSs and their acronyms.

File: drafts/Citing GA4GH Documentation.md [hunk 13]
```diff
@@ -0,0 +1,159 @@
...
+
+### Metadata Management
+
+Metadata associated with a DOI MUST accurately describe the referenced document at the time of registration. Metadata SHOULD be updated when document locations or other deposited metadata elements change. Where a document is relocated, the DOI MUST continue to resolve to the authoritative version of the document. Changes in document location MUST NOT require minting a new DOI unless a new citable object is being created.
+
+The complete list of required metadata follows. Fields marked *"to be specified"* MUST be supplied by the requester:
+
+- Contributor → Organization: to be specified (usually a GA4GH Work Stream)
```
Comment (monicacecilia) [inline drafts/Citing GA4GH Documentation.md]: Is this "Contributor" or "Organization?" It reads like the "Contributor" can be an "Organization," but please clarify.

File: drafts/Citing GA4GH Documentation.md (hunk 5 above)
Comment (monicacecilia) [inline drafts/Citing GA4GH Documentation.md]: Please add an example to help requesters prepare these fields.

File: drafts/Citing GA4GH Documentation.md [hunk 14]
```diff
@@ -0,0 +1,159 @@
...
+- Title: to be specified
+- Approval Date (month, day, year): to be specified
+- Publisher Name: Global Alliance for Genomics and Health
+- Publisher Place: to be specified
+- Standards Body Name: Global Alliance for Genomics and Health
+- Standards Body Acronym: GA4GH
+- Resource (URL): to be specified
+- Depositor Name (submitter): GA4GH (or GA4GH DOI program)
```
Comment (monicacecilia) [inline drafts/Citing GA4GH Documentation.md]: Are we still deciding what this should be? I'd say pick one and stick with it. Unless there is a clear use case when the "GA4GH DOI program" would not be the submitter.

File: drafts/Citing GA4GH Documentation.md [hunk 15]
```diff
@@ -0,0 +1,159 @@
...
+- Registrant: GA4GH (i.e. legal entity)
+- Submitter email address: Technical Team (ga4gh-tech-team@ga4gh.org)
```
Comment (monicacecilia) [inline drafts/Citing GA4GH Documentation.md]: How are we capturing these metadata? Is this meant to be computable? If so, is a title/entity name permissible in an "email" field or should we just keep the address?

File: drafts/Citing GA4GH Documentation.md [hunk 16]
```diff
@@ -0,0 +1,159 @@
...
+
+## Use Cases
+
+### Use case - Approved GA4GH Products
+
+When a product is approved through the GA4GH Product Development and Approval Process, DOIs MUST be assigned to support persistent and version-specific citation. At a minimum, DOI registration SHOULD be considered for:
```
Comment (monicacecilia) [inline drafts/Citing GA4GH Documentation.md]: What about the documentation pages? Several GA4GH products have GH pages or RTD pages with documentation. Their contents are usually found in the accompanying repository, but these pages represent the most comprehensive "documentary" elements of the standard/product. See, for example, documentation pages for VRS https://vrs.ga4gh.org/en/2.0/ and PXF https://phenopacket-schema.readthedocs.io/en/latest/. Does the DOI of the "approved version of the product specification point to the `product-schema.extension` file or to the entire`github.com/ga4gh/repository`?

File: drafts/Citing GA4GH Documentation.md [hunk 17]
```diff
@@ -0,0 +1,159 @@
...
+- Citation and machine-readable identification are distinct requirements. While a DOI provides a stable citation mechanism for publications and documentation, additional identifier mechanisms MAY be required where standards need to be referenced within APIs, service registries, metadata models, or software implementations.
+- DOI assignment SHOULD be considered independently from publication venue. A document MAY be published through multiple channels while retaining a single authoritative GA4GH-managed DOI.
+- Work Streams SHOULD establish clear guidance regarding the versioning strategy for documentary outputs. Readers and implementers MUST be able to determine which version of a specification is being referenced by a citation.
+- Multiple citation targets MAY exist for a single GA4GH product, including an approved specification, the latest specification, supporting documentation, and the product landing page. Work Streams SHOULD ensure that each DOI resolves to a clearly defined and distinguishable citable object.
```
Comment (monicacecilia) [inline drafts/Citing GA4GH Documentation.md]: I think this starts to resolve some of the questions I raised above. But good to discuss, because I'm already halfway through the docs and only now reading this for the first time. 😉

File: recommendations/Citing GA4GH Documentation.md [hunk 18]
```diff
@@ -0,0 +1,159 @@
...
+| James Eddy | Sage Bionetworks |
+| Ian Fore | Independent |
+| Francesca Frexia | Centre for Advanced Studies, Research and Development in Sardinia (CRS4) |
+| Jimmy Payyappilly | EMBL-EBI, GA4GH |
+| Angela Page | Broad Institute, GA4GH |
+| Alex Wagner | Nationwide Children's Hospital |
+| Andy Yates | HDR UK |
+| Michael Baudis | University of Zurich |
+| Sasha Siegel | EMBL-EBI, GA4GH |
+
```
Comment (monicacecilia) [inline recommendations/Citing GA4GH Documentation.md:201]: Dunno if you want me to fork and PR, but please add: 
`| Mónica Muñoz Torres | University of Colorado Anschutz |`

File: recommendations/Citing GA4GH Documentation.md [hunk 19]
```diff
@@ -0,0 +1,159 @@
...
+CLINPHEN_PHEN
+DISCOVERY_BEACONV2
+DSDURI_AAI
+TASC_TS
+```
+
+Note that over time, workstreams can change names. Use the name of the work stream at the time of minting. Work Streams SHOULD use concise and stable designator values to promote consistency across DOI registrations. The `<std_designator>` value MUST uniquely identify the document category within the context of the originating Work Stream.
+
+
+### DOI Request Process
```
Comment (monicacecilia) [inline recommendations/Citing GA4GH Documentation.md:76]: A workflow diagram here would be helpful. See attached example (AI-generated using Gemini, manually edited in Google Slides).

Image

**Review by monicacecilia — COMMENTED** [review]
Thank you for preparing this! It's brilliant, clear, concise, and it efficiently guides submitters through the process. I added a few comments and questions. Please see also: 
https://github.com/ga4gh/TASC/pull/86/changes#r3549419424
https://github.com/ga4gh/TASC/pull/86/changes#r3549441392
https://github.com/ga4gh/TASC/pull/86/changes#r3549445531
https://github.com/ga4gh/TASC/pull/86/changes#r3549453227
https://github.com/ga4gh/TASC/pull/86/changes#r3549460552
https://github.com/ga4gh/TASC/pull/86/changes#r3549501533
https://github.com/ga4gh/TASC/pull/86/changes#r3549510449

File: drafts/TASC-ADR-004.md [hunk 20]
```diff
@@ -0,0 +1,95 @@
...
+  ```
+
+Standard designators SHOULD remain stable over the lifetime of the referenced documentary output and therefore link items together.
+
+Examples:
+
+```text
+TASC_TS
+CLOUD_WES
```
Comment (mcupak) [inline drafts/TASC-ADR-004.md]:
```suggestion
FEDANALYSIS_WES
```

File: drafts/TASC-ADR-003.md [hunk 21]
```diff
@@ -0,0 +1,96 @@
...
+**Risk:** Users attempt to infer meaning from DOI values
+- **Mitigation:** Publish guidance stating that meaning is carried through metadata and standard designators rather than the DOI suffix
+
+
+---
+
+## References
+
+- **Full Policy:** [Citing GA4GH Documentation Recommendation](../drafts/Citing%20GA4GH%20Documentation.md)
+- **Related ADRs:** [TASC-ADR-001: Adopt a GA4GH DOI-Based Citation Framework](./TASC-ADR-001.md)
```
Comment (mcupak) [inline drafts/TASC-ADR-003.md]:
```suggestion
- **Related ADRs:** [TASC-ADR-001: Use Crossref as the DOI Registration Authority for GA4GH](./TASC-ADR-001.md)
```

File: drafts/Citing GA4GH Documentation.md [hunk 22]
```diff
@@ -0,0 +1,159 @@
...
+
+## References
+
+- \[CROSSREF\] \- [Crossref: a DOI issuer](https://www.crossref.org/)
+- \[CROSSREF_RECORDS\] \- [Crossref Supported Record Types](https://www.crossref.org/documentation/schema-library/markup-guide-record-types/)
+- \[TASC-ADR-001\] - [TASC-ADR-001: Use Crossref as the DOI Registration Authority for GA4GH](../adr/TASC-ADR-001.md)
+- \[TASC-ADR-002\] - [TASC-ADR-002: Separate Citation Identifiers from Machine-Readable Standard Identifiers](../adr/TASC-ADR-002.md)
+- \[TASC-ADR-003\] - [TASC-ADR-003: Use Opaque DOI Suffixes](../adr/TASC-ADR-003.md)
+- \[TASC-ADR-004\] - [TASC-ADR-004: Use Standard Designators for DOI Metadata Classification](../adr/TASC-ADR-004.md)
+- \[TASC-ADR-005\] - [TASC-ADR-005: Do Not Extend GA4GH DOIs to Datasets or Other Research Resources](../adr/TASC-ADR-005.md)
```
Comment (mcupak) [inline drafts/Citing GA4GH Documentation.md]:
```suggestion
- \[TASC-ADR-005\] - [TASC-ADR-005: Do Not Extend GA4GH DOIs to Other Research Resources](../adr/TASC-ADR-005.md)
```

File: recommendations/Citing GA4GH Documentation.md [hunk 23]
```diff
@@ -0,0 +1,159 @@
...
+- The creation of a DOI introduces a long-term maintenance obligation. GA4GH MUST ensure that associated metadata and resolution targets remain current throughout the lifetime of the identifier.
+- Changes to document hosting platforms, repository structures, website implementations, or publication locations MUST NOT result in unnecessary DOI reassignment. Existing DOIs SHOULD continue to resolve to the authoritative version of the referenced document.
+- Consistency across Work Streams is a primary objective of this recommendation. Work Streams SHOULD follow a common DOI registration and metadata model to avoid the introduction of competing citation practices within GA4GH.
+- DOI assignment improves citation stability but does not replace the need for product governance, version management, release processes, or publication review procedures.
+- Work Streams SHOULD consider the granularity of DOI assignment carefully. Assigning too few identifiers may limit citation precision, while assigning too many may introduce unnecessary governance and maintenance overhead.
+
+## References
```
Comment (mcupak) [inline recommendations/Citing GA4GH Documentation.md:173]: Links in this section point to `adr` directory (mismatch vs. `drafts`) - was the intent to call it `adr` and thus we should rename, or are they intentionally in a draft state?

File: drafts/Citing GA4GH Documentation.md [hunk 24]
```diff
@@ -0,0 +1,159 @@
...
+**Authors**: Chen Chen, James Eddy, Ian Fore, Francesca Frexia, Jimmy Payyappilly, Angela Page, Alex Wagner, Andy Yates, Michael Baudis, Sasha Siegel  
+**Date:** 2026-07-07  
+**Status:** Draft  
+**Keywords**: doi, citation  
+**Work Streams Impacted**: All workstreams  
+**Products Affected**: None directly  
+
+## Abstract
+
+This recommendation provides a uniform and consistent way for citing GA4GH products and documentary outputs by generation of a digital object identifier (DOI) where the GA4GH DOI prefix is `10.59756`. This guide is written from the understanding that DOIs will be minted via the Crossref service. We follow recommendations from Crossref \[CROSSREF\] to create DOIs whose suffixes are obfuscated and guidance on the required metadata to provide. The recommendation applies to GA4GH documentary outputs, including products, standards, protocols, technical specifications, white papers, and selected blog posts. It does not apply to datasets, tools, or other research resources.
```
Comment (mcupak) [inline drafts/Citing GA4GH Documentation.md]:
```suggestion
This recommendation provides a uniform and consistent way for citing GA4GH products and documentary outputs by generation of a digital object identifier (DOI) where the GA4GH DOI prefix is `10.59756`. This guide is written from the understanding that DOIs will be minted via the Crossref service. We follow recommendations from Crossref \[CROSSREF\] to create DOIs whose suffixes are opaque and guidance on the required metadata to provide. The recommendation applies to GA4GH documentary outputs, including products, standards, protocols, technical specifications, white papers, and selected blog posts. It does not apply to datasets, tools, or other research resources.
```

File: drafts/Citing GA4GH Documentation.md [hunk 25]
```diff
@@ -0,0 +1,159 @@
...
+- [Detailed Guidance](#detailed-guidance)
+- [Use Cases](#use-cases)
+- [Considerations](#considerations)
+- [References](#references)
+- [Contributors](#contributors)
+- [Notes](#notes)
+
+## Recommendation
+
+For significant GA4GH documentary outputs (e.g. products, standards, protocols, technical specifications, white papers, and selected blog posts) requiring a persistent identifier, Digital object identifiers (DOI) MUST use the GA4GH prefix `10.59756` resulting in DOIs with the following format `https://doi.org/10.59756/SUFFIX`. Registration requests MUST be submitted to the originating group's co-leads (e.g. work stream, NIF, GIF, TASC) and then to the Technical Team unless another automated minting method has been created. DOIs MUST be registered using the GA4GH approved Registration Agency, currently Crossref. DOI suffixes MUST be an obfuscated random string with no semantic meaning. The *Detailed Guidance* lists the minimal metadata that MUST be provided, which include the `<std_designator>` metadata field, pointing to the reference Work Stream.
```
Comment (mcupak) [inline drafts/Citing GA4GH Documentation.md]:
```suggestion
For significant GA4GH documentary outputs (e.g. products, standards, protocols, technical specifications, white papers, and selected blog posts) requiring a persistent identifier, Digital object identifiers (DOI) MUST use the GA4GH prefix `10.59756` resulting in DOIs with the following format `https://doi.org/10.59756/SUFFIX`. Registration requests MUST be submitted to the originating group's co-leads (e.g. work stream, NIF, GIF, TASC) and then to the Technical Team unless another automated minting method has been created. DOIs MUST be registered using the GA4GH approved Registration Agency, currently Crossref. DOI suffixes MUST be an opaque random string with no semantic meaning. The *Detailed Guidance* lists the minimal metadata that MUST be provided, which include the `<std_designator>` metadata field, pointing to the reference Work Stream.
```

File: drafts/TASC-ADR-002.md [hunk 26]
```diff
@@ -0,0 +1,78 @@
...
+**Deciders:** TASC, Technical Team, Architecture Community  
+**Keywords:** identifiers, citation, interoperability, APIs, governance  
+**Work Streams Impacted:** All  
+**Products Affected:** DRS, TRS, WES, TES, Beacon, VRS and future standards
+
+---
+
+## Context
+
+During discussions around standard citation, concerns were raised regarding the use of identifiers within APIs, schemas, registries, and machine-readable metadata
```
Comment (mcupak) [inline drafts/TASC-ADR-002.md]:
```suggestion
During discussions around standard citation, concerns were raised regarding the use of identifiers within APIs, schemas, registries, and machine-readable metadata.
```

File: drafts/TASC-ADR-002.md [hunk 27]
```diff
@@ -0,0 +1,78 @@
...
+3. **Separate citation and technical identification concerns**
+   - ✅ Clear governance boundaries
+   - ✅ Preserves future architectural flexibility
+   - ✅ Supports multiple technical use cases
+
+---
+
+## Decision
+
+GA4GH will use DOIs for documentary citation only. Future machine-readable identifier schemes MAY be developed independently where required for APIs, service registries, metadata models, and protocol implementations
```
Comment (mcupak) [inline drafts/TASC-ADR-002.md]:
```suggestion
GA4GH will use DOIs for documentary citation only. Future machine-readable identifier schemes MAY be developed independently where required for APIs, service registries, metadata models, and protocol implementations.
```

**Review by mcupak — COMMENTED** [review]
Very thorough and well argued!

Should we include "how to cite" guidance as suggested in https://github.com/ga4gh/TASC/issues/39#issuecomment-1171128966?

File: drafts/Citing GA4GH Documentation.md (hunk 2 above)
Comment (mcupak) [reply drafts/Citing GA4GH Documentation.md]: +1 since this can also include groups that are not their own work stream, e.g. TASC.

File: drafts/Citing GA4GH Documentation.md [hunk 28]
```diff
@@ -0,0 +1,159 @@
...
+
+The `<std_designator>` metadata field SHOULD follow the format `<WORK_STREAM_ACRONYM>_<DOCUMENT_NAME>`.
+
+The Technical Team maintains a register of assigned suffixes to avoid clashing DOIs. DOIs MAY be created for any GA4GH digital output and for any version of a digital output. Newly approved products SHOULD request and assign a DOI to selected digital materials, including any specific versions. DOIs MUST NOT be minted for transitional or ephemeral materials e.g. a draft specification. 
+
+GA4GH DOIs work alongside those created by other authorities and organisations e.g. publishers, open repositories and do not replace or supplant them. GA4GH DOIs MUST NOT redirect to another DOI URI. This recommendation MUST apply to GA4GH documentary outputs and MUST NOT apply to datasets, tools or other research resources.
```
Comment (ahwagner) [inline drafts/Citing GA4GH Documentation.md]:
```suggestion
GA4GH DOIs work alongside those created by other authorities and organisations (e.g. publishers, open repositories) and do not replace or supplant them. GA4GH DOIs MUST NOT redirect to another DOI URI. This recommendation MUST apply to GA4GH documentary outputs and MUST NOT apply to datasets, tools or other research resources.
```

**Review by ahwagner — APPROVED** [review]

File: drafts/Citing GA4GH Documentation.md (hunk 25 above)
Comment (TomConner) [inline drafts/Citing GA4GH Documentation.md]:
```suggestion
For significant GA4GH documentary outputs (e.g. products, standards, protocols, technical specifications, white papers, and selected blog posts) requiring a persistent identifier, Digital object identifiers (DOI) MUST use the GA4GH prefix `10.59756` resulting in DOIs with the following format `https://doi.org/10.59756/SUFFIX`. Registration of DOIs MUST be registered using the GA4GH approved Registration Agency, currently Crossref. DOI suffixes MUST be an obfuscated random string with no semantic meaning. The *Detailed Guidance* lists the minimal metadata that MUST be provided, which include the `<std_designator>` metadata field, pointing to the reference Work Stream.

Registration requests MUST be submitted to the originating group's co-leads (e.g. work stream, NIF, GIF, TASC) and then to the Technical Team as TASC issue as a separate unless another automated minting method has been created. 
```

Separated the request process and specified opening a TASC issue the correct way to submit the request. Perhaps the submission process is not a TASC issue, but I suggest it be specified.

**Review by TomConner — CHANGES_REQUESTED** [review]

File: recommendations/Citing GA4GH Documentation.md (hunk 18 above)
Comment (rrfreimuth) [reply recommendations/Citing GA4GH Documentation.md:201]: Ditto, please.
`|Robert R. Freimuth | Mayo Clinic |`

File: drafts/Citing GA4GH Documentation.md (hunk 17 above)
Comment (monicacecilia) [reply drafts/Citing GA4GH Documentation.md]: During the TASC workshop discussion, we decided that DOIs will be minted for at least the following: 
- the latest version of the product
- the product's landing page on the GA4GH website
- the product's documentation (e.g., GitHub Pages, ReadTheDocs, etc.)

File: drafts/Citing GA4GH Documentation.md (hunk 4 above)
Comment (andy-yates-hdr) [reply drafts/Citing GA4GH Documentation.md]: Clarification needed from Crossref

File: drafts/Citing GA4GH Documentation.md (hunk 5 above)
Comment (andy-yates-hdr) [reply drafts/Citing GA4GH Documentation.md]: Same this is Crossref. Check

File: drafts/Citing GA4GH Documentation.md (hunk 4 above)
Comment (andy-yates-hdr) [reply drafts/Citing GA4GH Documentation.md]: Approval date of document not of submission to Crossref

File: drafts/Citing GA4GH Documentation.md (hunk 6 above)
Comment (andy-yates-hdr) [reply drafts/Citing GA4GH Documentation.md]: Yes

File: drafts/Citing GA4GH Documentation.md (hunk 4 above)
Comment (larrybabb) [reply drafts/Citing GA4GH Documentation.md]: > The key distinction is:
> 
> **approval_date** = formal scholarly/institutional approval of the work itself
> **publication_date** = when the work was published or made available
> **created / deposited / indexed dates** = Crossref system dates about DOI registration or metadata processing


tl;dr

see https://chatgpt.com/s/t_6a4f7ce7abb08191bd724b6a430bdc89 to understand the dates that Crossref generates

File: adr/TASC-ADR-002.md (hunk 7 above)
Comment (andy-yates-hdr) [reply adr/TASC-ADR-002.md:67]: Who is responsible and where should it live.

- Linked to the product?
- Linked to the tech team?
- GA4GH website

Both product and listing for tech team

File: drafts/Citing GA4GH Documentation.md (hunk 12 above)
Comment (andy-yates-hdr) [reply drafts/Citing GA4GH Documentation.md]: Need that list of acronyms

File: drafts/Citing GA4GH Documentation.md (hunk 13 above)
Comment (andy-yates-hdr) [reply drafts/Citing GA4GH Documentation.md]: Lets just say GA4GH

File: drafts/Citing GA4GH Documentation.md (hunk 14 above)
Comment (andy-yates-hdr) [reply drafts/Citing GA4GH Documentation.md]: Example needed

File: drafts/TASC-ADR-004.md (hunk 20 above)
Comment (andy-yates-hdr) [reply drafts/TASC-ADR-004.md]: Advise use of CFF

File: recommendations/Citing GA4GH Documentation.md (hunk 19 above)
Comment (andy-yates-hdr) [reply recommendations/Citing GA4GH Documentation.md:76]: Diagram now in mermaid

File: recommendations/Citing GA4GH Documentation.md (hunk 23 above)
Comment (andy-yates-hdr) [reply recommendations/Citing GA4GH Documentation.md:173]: Noted this wil resolve once merged properly

File: adr/TASC-ADR-002.md (hunk 7 above)
Comment (andy-yates-hdr) [reply adr/TASC-ADR-002.md:67]: Added

File: adr/TASC-ADR-003.md (hunk 8 above)
Comment (andy-yates-hdr) [reply adr/TASC-ADR-003.md:82]: Done

File: adr/TASC-ADR-004.md (hunk 10 above)
Comment (andy-yates-hdr) [reply adr/TASC-ADR-004.md:88]: Done

File: adr/TASC-ADR-005.md (hunk 11 above)
Comment (andy-yates-hdr) [reply adr/TASC-ADR-005.md:41]: Done

**Comment by andy-yates-hdr** [comment]
Okay all changes integrated in. @TomConner there is a re-review requested from you since you asked for changes and these were applied but not picked up on by GitHub

**Comment by andy-yates-hdr** [comment]
- All issues raised have been resolved
- New diagram developed from @monicacecilia (she/ella) input
- New registry of prefixes generated 
- Files are now moved out of drafts into their correct locations

**Review by TomConner — APPROVED** [review]

File: adr/TASC-ADR-004.md [hunk 29]
```diff
@@ -0,0 +1,103 @@
...
+
+Standard designators SHOULD remain stable over the lifetime of the referenced documentary output and therefore link items together.
+
+Examples:
+
+```text
+TASC_TS
+FEDANALYSIS_WES
+CLINPHENO_PHEN
+DISCOVERY_BEACONV2
```
Comment (mbaudis) [inline adr/TASC-ADR-004.md:66]: Not happy about `DISCOVERY_BEACONV2`; should be `DISCOVERY_BEACON` (we're trying to discourage inlining versions in the name...).

Comment (andy-yates-hdr) [reply adr/TASC-ADR-004.md:66]: Missed this sorry! Done
